prow/plugins/dco/dco.go

```diff
@@ -97,6 +117,32 @@ type commentPruner interface {
 	PruneComments(shouldPrune func(github.IssueComment) bool)
 }
 
+// checkTrustedUser checks are all commits from a trusted user
+func checkTrustedUser(dco plugins.Dco, gc gitHubClient, l *logrus.Entry, org, repo string, number int) (bool, error) {
+	// Skip checking is user trusted if this option isn't enabled.
+	if !dco.SkipDcoCheck {
+		return false, nil
+	}
+
+	allCommits, err := gc.ListPRCommits(org, repo, number)
+	if err != nil {
+		return false, fmt.Errorf("error listing commits for pull request: %v", err)
+	}
+
+	for _, commit := range allCommits {
+		trusted, err := trigger.TrustedUser(gc, dco.OnlyOrgMembers, dco.TrustedOrg, commit.Author.Login, org, repo)
+		if err != nil {
+			return false, fmt.Errorf("Error checking is member trusted: %v", err)
+		}
+		if !trusted {
+			l.Debugf("Member %s is neither org member or repo collaborator", commit.Author.Login)
```

Comment:
Can we just say that the member is not trusted here? If `onlyOrgMembers` is true, I'm not sure we can accurately say if the commit author is not a repo collaborator...

Reply:
Fixed